### PR TITLE
Add new API endpoint to give all fleets in an airport.

### PR DESCRIPTION
**Pull request by vgpastor**
No other methods return this info, only it's possible return all fleets and filter it, but this when we have hundreds of fleets it's very slow.

### CODE REVIEW

**Comment by nabeelio** [comment]
Hey, this looks good - just a couple of changes - can you move this into the `AirportController` and call the method `fleet`? And change the endpoint to `/airport/{id}/fleet`, since the airport is the main thing here. Thanks!

File: app/Http/Controllers/Api/FleetController.php [hunk 1]
```diff
@@ -69,4 +69,38 @@ public function get_aircraft($id, Request $request)
...
+     * @param Request $request
+     *
+     * @return AircraftResource
+     */
+    public function get_airport($id, Request $request)
+    {
+        $aircrafts = $this->aircraftRepo
+            ->with(['subfleet', 'subfleet.fares', 'subfleet.ranks']);
+
+        if ($request->filled('rank')) {
```
Comment (nabeelio) [inline app/Http/Controllers/Api/FleetController.php:88]: Is this to determine for a user's rank, or any specific rank?

Comment (vgpastor) [reply app/Http/Controllers/Api/FleetController.php:88]: This allow any idranks, because it's possible that aircraft are not restricted for ranks, or that only need show statics or other data about airports.

**Comment by vgpastor** [comment]
Hi, i updated the function to move the code to Airport Controller.

**Comment by vgpastor** [comment]
TRAVIS said that test fails, but in m localwork nice, anyone can run and confirm me that are failing?

File: tests/ApiTest.php [hunk 2]
```diff
@@ -283,6 +286,13 @@ public function testGetAircraft()
         $this->assertEquals($body['name'], $aircraft->name);
         $this->assertEquals($body['mtow'], $aircraft->mtow);
         $this->assertEquals($body['zfw'], $aircraft->zfw);
+        $resp = $this->get('/api/airports/'.$this->user->curr_airport_id.'/fleet');
```
Comment (nabeelio) [inline tests/ApiTest.php:289]: Put this in a new test (`testGetAircraftAtAirport` or something), you can copy/paste it over but it's testing different functionality. Thank you for adding it into the tests, though

Comment (vgpastor) [reply tests/ApiTest.php]: Done!
In the future this test can be extended to test parameters and rank of the returned aircrafts.

**Comment by nabeelio** [comment]
If this is being used to filter on the PIREP file page, then there should be a filter there also based on the correct setting of whether those aircraft are filtered or not

**Comment by vgpastor** [comment]
> If this is being used to filter on the PIREP file page, then there should be a filter there also based on the correct setting of whether those aircraft are supposed to be filtered or not

Can you explain me this? i can't understand what do you referer

**Comment by vgpastor** [comment]
> > If this is being used to filter on the PIREP file page, then there should be a filter there also based on the correct setting of whether those aircraft are supposed to be filtered or not
> 
> Can you explain me this? i can't understand what do you referer

I thinks that this upgrade can be added in new versions, that can be made in this PR or in others

File: tests/SimBriefTest.php [hunk 3]
```diff
@@ -97,10 +97,10 @@ public function testApiCalls()
         $this->assertNotNull($flight['simbrief']);
         $this->assertEquals($briefing->id, $flight['simbrief']['id']);
 
-        $url = str_replace('http://', 'https://', $flight['simbrief']['url']);
-        $this->assertEquals(
-            'https://localhost/api/flights/'.$briefing->flight_id.'/briefing',
```
Comment (nabeelio) [inline tests/SimBriefTest.php:102]: What's the reason for this change?

Comment (vgpastor) [reply tests/SimBriefTest.php:102]: When i make the test in a local php session the server are 127.0.0.1:XXXX and not in localhost, i think that if we verify the url path and not the host the test evaluate the same
